docs/core/macos-prerequisites.md

```diff
@@ -45,32 +24,30 @@ See [.NET Core 3.0 Supported OS Versions](https://github.com/dotnet/core/blob/ma
 
 For download links and more information, see [.NET Core 3.0 downloads](https://dotnet.microsoft.com/download/dotnet-core/3.0).
 
----
+# [.NET Core 2.x](#tab/netcore2x)
```

Comment:
2.1 is EOL. Perhaps change this to tab to **.NET Core 2.2:** and remove links to .NET Core 2.1 information.

Reply:
Is 2.1 really EOL? <https://dotnet.microsoft.com/platform/support/policy/dotnet-core> says .NET Core 2.2 end of support will be on December 23, 2019, but .NET Core 2.1 will last "At least three years from LTS declaration (August 21, 2018)".

Reply:
@KalleOlaviNiemitalo no it was just a mistake! 😄